src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -827,8 +827,8 @@ protected virtual void Dispose(bool disposing)
                     {
                         _hotkeyManager.UnregisterHotkey(_hotkeyHandle);
```

Comment:
@enricogior, @ivan100sic,  I understand that this is already merged in but I see that we're doing a null check for `_hotkeyManager` only for Dispose and not for `UnregisterHotKey`. If the issue was caused due to the `_hotkeyManager` being nul (not _updateSource), it would still persist.

Reply:
@alekhyareddy28 
thanks for pointing it out, feel free to open a new PR if you have the time, otherwise we can do it tomorrow.